### ISSUE TITLE
MiniSplit Heat Mode 

### ISSUE THREAD

**Issue by scaryfast8750** [kovapatrik/homebridge-midea-platform#80]
On my MiniSplit using the Midea app I can choose Cool, Heat, Dry, Auto Fan.

Using the plugin only Off, Cool and Auto are available.  No Fan or Heat mode.  I saw a previous feature request where you fixed that but I dont see it in my config.  Can we get Heat support added?

Note:  If I put the split in Heat Mode using the Midea App, Heat mode shows correctly in HomeKit.  See screenshots below.

![Screenshot](https://github.com/kovapatrik/homebridge-midea-platform/assets/88514039/f844d971-108b-4d11-864b-5fc8d8b862d5)
![image](https://github.com/kovapatrik/homebridge-midea-platform/assets/88514039/8f8a9ec8-e44a-4a2a-9e92-4b1e1a2572a2)

**Comment by kovapatrik** [comment]
Please read the docs about configuring your devices. From the Homebridge UI you can set if your AC is heat capable and also enable fan-only mode switch.

**Comment by scaryfast8750** [comment]
@kovapatrik Definitely went through the docs in detail.  I have that enabled...here is my setting.

![image](https://github.com/kovapatrik/homebridge-midea-platform/assets/88514039/a2efa435-0eb9-4519-9e4e-d7d08ce1804f)

**Comment by kovapatrik** [comment]
Did you restart the child bridge (or Homebridge itself) after setting it?

**Comment by scaryfast8750** [comment]
It was already set, and Yes I did restart both the child bridge and entire Homebridge platform after. 

I just did the following steps:
Remove Hub from Homekit
Restart Homebridge platform (includes child bridges obviously)
Re-Add to HomeKit with QR
Bridge/Accessory successfully added.

Here are the log file errors which dont appear to be related:  [4/11/2024, 2:28:53 PM] [homebridge-midea-platform] This plugin generated a warning from the characteristic 'Rotation Speed': characteristic was supplied illegal value: number 999 exceeded maximum of 102. See https://homebridge.io/w/JtMGR for more info.

Now Heat Mode is available.  I believe the Delete/Re-Add is what fixed it.  I think I had added the accessory before I finished configuring in HomeBridge.
![image](https://github.com/kovapatrik/homebridge-midea-platform/assets/88514039/9eafbadb-a203-41fc-a44a-dc41ea3a4d65)


My apologies for burdening you with this.
